lib/libdl/dlfcn.c

```diff
@@ -0,0 +1,125 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2019 Linaro limited
+ */
+
+#include <ctype.h>
+#include <dlfcn.h>
+#include <pta_system.h>
+#include <stdlib.h>
+#include <string.h>
+#include <tee_api.h>
+#include <tee_internal_api_extensions.h>
+
+static TEE_TASessionHandle sess = TEE_HANDLE_NULL;
+
+static TEE_Result invoke_system_pta(uint32_t cmd_id, uint32_t param_types,
+				    TEE_Param params[TEE_NUM_PARAMS])
+{
+	const TEE_UUID core_uuid = PTA_SYSTEM_UUID;
+	TEE_Result res = TEE_ERROR_GENERIC;
+
+	if (!sess) {
+		res = TEE_OpenTASession(&core_uuid, 0, 0, NULL, &sess, NULL);
+		if (res)
+			return res;
+	}
+	res = TEE_InvokeTACommand(sess, 0, cmd_id, param_types, params, NULL);
```

Comment:
minor: `return TEE_InvokeTACommand(...` (fits in 80 chars :)

Reply:
ok